internal/consul/consul.go

```diff
@@ -39,7 +41,15 @@ type Health interface {
 }
 
 func DefaultClient() (Client, error) {
-	w, err := consulApi.NewClient(consulApi.DefaultConfig())
+	config := consulApi.DefaultConfig()
+	addrVal, addrPresent := os.LookupEnv("CONSUL_ADDR")
+	if addrPresent {
+		config.Address = addrVal
+	} else {
+		config.Address = "127.0.0.1:8500"
+	}
```

Comment:
I don't think we need this `else` statement, since that's actually the default value, as shown here:
https://github.com/hashicorp/consul/blob/8d6cbe72813779f60ac24d119cc311c21003f4ce/api/api.go#L396

Reply:
Totally right, I'll remove it.